### PR TITLE
Add cards disappear on guessing. Closes: #8

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

**Comment by probil** [comment]
@liubich try to *rebase* it on the latest *master*. 
Rebase is useful technique to keep old branches up-to-date with latest changes

**Comment by liubich** [comment]
@probil, rebase is made

**Comment by probil** [comment]
@liubich Nice :+1: 
But refactor with arrow function still needed :)

File: main.js [hunk 1]
```diff
@@ -39,11 +39,15 @@ class Game {
             this.prevClicked = idClicked;
         }
         else {
-            if(this.imagesRandomised[idClicked] == this.imagesRandomised[this.prevClicked]) {
-                this.imageContainers[idClicked].removeEventListener('click', this.imageOnClickBinded);
-                this.imageContainers[this.prevClicked].removeEventListener('click', this.imageOnClickBinded);
-                this.prevClicked = undefined;
-                this.nowOpened = 0;
+            if(this.imagesRandomised[idClicked] === this.imagesRandomised[this.prevClicked]) {
+                setTimeout(function(t) {
```
[the remark this answers is not in the document]
Comment (liubich) [reply main.js]: @probil, thank you

**Pull request merged**